### PR TITLE
Kuz 335 list realtime collections

### PR DESCRIPTION
**Pull request by scottinet**
* implemented the possibility to get realtime collections using the existing "listCollections" API method
* updated unit tests, functional tests and API documentation
* tidied up the ``readController`` unit tests
* removed remnants of the old ``roomName`` argument in the ``hotelClerk.removeCustomerFromAllRooms`` unit tests
* added missing unit tests on the ``removeRooms`` method of the ``adminController``

### CODE REVIEW

**Comment by benoitvidis** [comment]
+1
NB: taken into account on the new api doc already (in progress).

**Comment by scottinet** [comment]
@benoitvidis : wait before updating the new API documentation. Right now, this new request is the only one not taking its arguments from the document body, and this is wrong. I'm changing that.

**Comment by scottinet** [comment]
Fixed.

**Comment by ballinette** [comment]
+1

File: lib/api/controllers/readController.js [hunk 1]
```diff
@@ -23,9 +23,35 @@ module.exports = function ReadController (kuzzle) {
   };
 
   this.listCollections = function (requestObject) {
```
Comment (AnthonySendra) [inline lib/api/controllers/readController.js]: Here, we need to return only allowed collections

**Comment by AnthonySendra** [comment]
+1

**Pull request merged**